### PR TITLE
Add methods to handle new lines and components in ForgeI18n

### PR DESCRIPTION
**Pull request by DarkEyeDragon**
Would like some feedback since i'm not sure if this is the right way forward for this

### CODE REVIEW

File: src/main/java/net/minecraftforge/fml/ForgeI18n.java [hunk 1]
```diff
@@ -30,9 +32,7 @@
 import java.text.FieldPosition;
 import java.text.Format;
 import java.text.ParsePosition;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```
Comment (TheCurle) [inline src/main/java/net/minecraftforge/fml/ForgeI18n.java]: Bulk imports are annoying to maintain. Keep it concise.

**Review by TheCurle — APPROVED** [review]
Seems good. Minor niggles to adjust, but it's not important.

**Comment by sciwhiz12** [comment]
Hello! @DarkEyeDragon, please leave a comment whether you wish to continue this PR and update it to the latest active version of 1.18 (or leave an explanation why this PR's changes are not applicable to 1.18). If we do not hear back from you after some time, this PR will be closed for inactivity.
If anyone else wishes to continue the work of this PR, please create a PR with a link to this PR for reference. 
Thank you for your contributions. 👋

**Comment by DarkEyeDragon** [comment]
> Hello! @DarkEyeDragon, please leave a comment whether you wish to continue this PR and update it to the latest active version of 1.18 (or leave an explanation why this PR's changes are not applicable to 1.18). If we do not hear back from you after some time, this PR will be closed for inactivity. If anyone else wishes to continue the work of this PR, please create a PR with a link to this PR for reference. Thank you for your contributions. 👋

I haven't touched 1.18 all that much yet so I'm not sure if its still relevant. I'll do some research and if needed i'll port to 1.18 otherwise i'll close it

**Comment by DarkEyeDragon** [comment]
Seems like this is no longer an issue in 1.18.x
The server compat tooltip seem to be handling line endings just fine now.
image

Closes #7749

**Pull request closed without merging**